### PR TITLE
SDI-477: ✨ test moving message from queue to dlq

### PR DESCRIPTION
**Pull request by petergphillips**
- SDI-477: ✨ test moving message from queue to dlq

### CODE REVIEW

File: hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsProperties.kt [hunk 1]
```diff
@@ -20,6 +20,7 @@ data class HmppsSqsProperties(
     val dlqAccessKeyId: String = "",
     val dlqSecretAccessKey: String = "",
     val dlqMaxReceiveCount: Int = 5,
+    val visibilityTimeout: Int = 30,
```
Comment (petergphillips) [inline hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsProperties.kt:23]: default is 30 seconds in AWS, so duplicate that behaviour by default

File: test-app-reactive/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagenameasync/service/OutboundEventsEmitter.kt [hunk 2]
```diff
@@ -35,6 +34,6 @@ class OutboundEventsEmitter(hmppsQueueService: HmppsQueueService, private val ob
         )
         .build()
         .also { log.info("Published event $hmppsEvent to outbound topic") }
-    ).await()
```
Comment (petergphillips) [inline test-app-reactive/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagenameasync/service/OutboundEventsEmitter.kt:38]: think this was a cut and paste error from the non reactive version

File: hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsProperties.kt (hunk 1 above)
Comment (mikehalmamoj) [reply hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsProperties.kt:23]: Can we document this in the README? Something to say that this is only relevant for testing with Localstack...

**Review by steverendell — DISMISSED** [review]
LGTM

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**